pkg/cli-runtime/client.go

```diff
@@ -66,6 +72,19 @@ func (c *client) Discovery() discovery.DiscoveryInterface {
 func (c *client) Client() crclient.Client {
 	return c.lazyLoadClientOrDie()
 }
+func (c *client) ToRESTConfig() (*rest.Config, error) {
+	return c.KubeRestConfig(), nil
+}
+
+func (c *client) ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error) {
+	return disk.NewCachedDiscoveryClientForConfig(c.restConfig, "", "", 0) // need alignment with sash
+}
+func (c *client) ToRawKubeConfigLoader() clientcmd.ClientConfig {
+	return nil
```

Comment:
Is it fine to return `nil` here, is `ToRawKubeConfigLoader()`going to be used? if not why is it added to the interface?

Reply:
We are not using it and I have updated the code to remove it from interface as well as implementation